azurerm/data_source_servicebus_namespace.go

```diff
@@ -0,0 +1,105 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmServiceBusNamespace() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmServiceBusNamespaceRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"resource_group_name": resourceGroupNameForDataSourceSchema(),
+
+			"location": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"sku": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"capacity": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			"default_primary_connection_string": {
```

Comment:
Where does the term default come from? the SDK properties don't use it

Reply:
this information comes from the `RootManageSharedAccessKey` Authorization Rule - and is/was labelled as this in the portal at one time (and is why we use this in the resource, from memory)